pandas/core/indexes/datetimelike.py

```diff
@@ -430,8 +430,8 @@ def argmax(self, axis=None, skipna=True, *args, **kwargs):
     # --------------------------------------------------------------------
     # Rendering Methods
 
-    def _format_with_header(self, header, **kwargs):
-        return header + list(self._format_native_types(**kwargs))
+    def _format_with_header(self, header, na_rep='NaT', **kwargs):
```

Comment:
No big deal, but I think there's a typo (`'NaT'` instead of `'NaN'`). Or is it intentional?

Reply:
this is correct, NaT is the datetime repr for missing values.

Reply:
My bad!